client/src/App.js

```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+//import Modal from '@material-ui/core/Modal';
 import './App.css';
 import Header from './components/layout/header/Header';
 import Sidenav from './components/layout/sidenav/Sidenav';
 import Buildings from './components/pages/Buildings';
 import Companies from './components/pages/Companies';
 import Boilers from './components/pages/Boilers';
 import Technicians from './components/pages/Technicians';
+import BoilerTypes from './components/pages/Boiler-types/Boiler-types';
```

Comment:
Make sure that the files and folders names follow the good practices and conventions